### PR TITLE
Add contributing guidelines & PR template

### LINKED ISSUE #269: Proper style guide/contributing guidelines

### PR DESCRIPTION
**Pull request by ozkriff**
Closes #269 (_"Proper style guide/contributing guidelines"_)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,234 @@
...
+- Keep in mind that more than half of readers consume the newsletter
+  using mobile devices.
+  So try to avoid things that doesn't work well with small screens:
+  nested lists, long titles, images with important small details,
+  code blocks with long lines, etc.
+
+- Don't use fourth-level headers.
+  Divide a section into subsections using a `------` line if needed.
+
+- Avoid using blold, italic, etc rich formatting if possible.
```
Comment (17cupsofcoffee) [inline CONTRIBUTING.md:142]:
```suggestion
- Avoid using bold, italic, etc rich formatting if possible.
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,21 @@
...
-- A new draft for the next month is created from the template, goto 1.
+## Contributing Guide
+
+To contribute to the newsletter, please see [CONTRIBUTING].
+
+[CONTRIBUTING]: CONTRIBUTING.md
+
+## Building from Source
+
+The site is build and deployed automatically from the repo (see .github/workflows/ci.yml).
```
Comment (17cupsofcoffee) [inline README.md:11]:
```suggestion
The site is built and deployed automatically from the repo (see .github/workflows/ci.yml).
```

**Review by 17cupsofcoffee — APPROVED** [review]
Couple of minor typos, but really happy with the content, looks good 👍

**Comment by ozkriff** [comment]
:timer_clock: Merging in ~12 hours if there're no objections.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,234 @@
...
+
+[markdownlint]: https://github.com/DavidAnson/markdownlint
+[vscode-lint]: https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint
+[sublime-lint]: https://packagecontrol.io/packages/SublimeLinter-contrib-markdownlint
+[vim-lint]: https://github.com/fannheyward/coc-markdownlint
+[softbreak]: https://spec.commonmark.org/0.29/#soft-line-breaks
+[md-reflinks]: https://www.markdownguide.org/basic-syntax/#reference-style-links
+
+Ping the coordinators in the current coordination issue
+or WG's Discord channel if there're any questions.
```
Comment (Wodann) [inline CONTRIBUTING.md:232]:
```suggestion
or WG's Discord channel if there are any questions.
```

**Review by Wodann — CHANGES_REQUESTED** [review]
Looks good to me. Nice work! I just found some grammar mistakes.

**Review by Wodann — APPROVED** [review]

**Pull request merged**